### PR TITLE
Create xpr.json

### PR DESCRIPTION
**Pull request by alexpkgs**
created my xpr.json

### CODE REVIEW

**Comment by alexpkgs** [comment]
and also if i update my page at codeberg will it still be updated on the is-a.dev page of it?

File: domains/xpr.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "xprsh site",
+  "repo": "https://codeberg.org/xpr/pages",
+  "owner": {
+    "username": "xpr",
+    "email": "xpr@atl.tools",
+    "discord": "xprsh"
+  },
+  "record": {
+    "CNAME": "xprsh.codeberg.page"
```
Comment (Stef-00012) [inline domains/xpr.json:10]: I cannot react your website, 404
Codeberg says:
> "could not find a valid repository or branch for repository: "

**Review by Stef-DP — DISMISSED** [review]

**Comment by Stef-00012** [comment]
> and also if i update my page at codeberg will it still be updated on the is-a.dev page of it?

Yes, any update you apply on your host, will always be redirected to your is-a.dev domain

File: domains/xpr.json (hunk 1 above)
Comment (alexpkgs) [reply domains/xpr.json:10]: so do i need to add the branch in the url? y/n
or i have to use github pages?

**Comment by alexpkgs** [comment]
so i fixed it it was a typo

**Review by iostpa — APPROVED** [review]
Seems good to me

**Comment by alexpkgs** [comment]
@CuteDog5695 
image
theres a problem...

**Comment by CuteDog5695** [comment]
Did you try connecting the domain?

**Comment by alexpkgs** [comment]
no, but like how do i connect to the domain? @CuteDog5695

**Comment by alexpkgs** [comment]
okay so instead of using codeberg pages imma use github pages cuz it has the custom domain thingy

**Comment by alexpkgs** [comment]
edit:
 },
  "record": {
    "CNAME": "https://alexpkgs.github.io"

**Comment by CuteDog5695** [comment]
> no, but like how do i even connect to the domain? @CuteDog5695

Late message, but theres always [our documentation](https;//is-a.dev/docs) :/

**Pull request merged**